### ISSUE TITLE
"securityWorker" is only used if "secure" option is specified on each request

### ISSUE THREAD

**Issue by dkamyshov** [acacode/swagger-typescript-api#212]
Hi! Thanks for the great package.

This seems to be a Fetch API variation of another issue - https://github.com/acacode/swagger-typescript-api/issues/192.

Here is what the `request` method looks like after generating the API with the following parameters:

```sh
$ yarn swagger-typescript-api -p ./source.json -o . -n result.ts
```

```ts
class Api {
  // ...

  public request = async <T = any, E = any>({
    body,
    secure,
    // ...
    ...params
  }: FullRequestParams): Promise<HttpResponse<T, E>> => {
    // the "secure" option from this.baseApiParams is not used here
    const secureParams =
      (secure &&
        this.securityWorker &&
        (await this.securityWorker(this.securityData))) ||
      {};

    const requestParams = this.mergeRequestParams(params, secureParams);

    // ...
  };
}
```

So, basically, when the Fetch API is used, there is no way to make use of `securityWorker` other than specifying `secure` on each request. With `axios` this is solved trivially with interceptors, but I'd like to continue using the Fetch API.

Here is a simple reproduction of the issue (v7.0.0, just run `$ yarn`): https://github.com/dkamyshov/swagger-typescript-api-bug-repro

| swagger-typescript-api | 6.4.2, 7.0.0 |
| ---------------------- | ------------ |

**Comment by dkamyshov** [comment]
Replacing `secure` with `(secure ? true : this.baseApiParams.secure)` (no `??` for Node 12, ofc) [here](https://github.com/acacode/swagger-typescript-api/blob/b5d3d502206e9370d1a36add7fbf1eaae76501de/templates/base/http-clients/fetch-http-client.eta#L156) and [here](https://github.com/acacode/swagger-typescript-api/blob/8354a4fe57dac803272228558223ab6fce25c5a4/templates/base/http-clients/axios-http-client.eta#L72) is probably enough. I can open a PR if you approve this.

**Comment by js2me** [comment]
@dkamyshov please open a PR, I'll approve this.   
Please use the `next` branch

**Comment by js2me** [comment]
@dkamyshov thank you for your contribution!   
This problem should be fixed in 7.0.1

**Comment by dkamyshov** [comment]
@js2me thanks for quick response, it works flawlessly now.
